app/views/reminders/index.html.erb

```diff
@@ -1,33 +1,28 @@
 <p id="notice"><%= notice %></p>
```

Comment:
This page is not accessible from the navbar. Add a navlink to the user dropdown in application.html.erb